core/src/main/java/com/amplifyframework/api/ApiChannelEventName.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.api;
+
+import com.amplifyframework.hub.HubCategory;
+import com.amplifyframework.hub.HubChannel;
+import com.amplifyframework.hub.HubEvent;
+
+/**
+ * An enumeration of the names of events relating the the {@link ApiCategory},
+ * that are published via {@link HubCategory#publish(HubChannel, HubEvent)} on the
+ * {@link HubChannel#API} channel.
+ */
+public enum ApiChannelEventName {
+    /**
+     * The status of the remote API endpoint has changed.
```

Comment:
What kinds of status changes are these? Like, we are able to reach it or not?

Reply:
I did a little word smithing there. Let me know if it looks OK.